### ISSUE TITLE
Add a label below the search bar that says "DEVELOPER PREVIEW ONLY", to manage expectations

### ISSUE THREAD

**Issue by ctrueden** [imagej/napari-imagej#44]
We can remove the banner once we feel like napari-imagej has reached a state where most things work as expected.

**Comment by gselzer** [comment]
@ctrueden what if we instead renamed the widget? That would be an even more prominent place, and requires the change of only `napari.yml`:
![napari_developer_preview](https://user-images.githubusercontent.com/29754838/170518092-969027e1-93f9-47ba-be6a-c7535193ccde.png)

**Comment by ctrueden** [comment]
Yes, I like it. Very prominent, and less disruptive to the interface. Go for it.
